.buildkite/ftr_configs.yml

```diff
@@ -76,7 +76,6 @@ enabled:
   - test/functional/apps/dashboard/group3/config.ts
   - test/functional/apps/dashboard/group4/config.ts
   - test/functional/apps/dashboard/group5/config.ts
-  - test/functional/apps/dashboard/group6/config.ts
```

Comment:
ℹ️ The group5 was testing the dashboard with vislib pies. I removed the group and renamed group6 to group5